ServiceDocumentSampleComponents/serviceDocumentSampleComponents/README.md

```diff
@@ -0,0 +1,18 @@
+# Salesforce DX Project: Next Steps
+
+Now that you’ve created a Salesforce DX project, what’s next? Here are some documentation resources to get you started.
```

Comment:
We should make this readme relevant for service document users. At least a blurb at the top or something.

Reply:
Good point, I updated the readme doc.